qcodes/dataset/plotting.py

```diff
@@ -94,13 +95,13 @@ def heatmaphandler(**kwargs: Any) -> Any:
 
 def plot_dataset(
     dataset: DataSetProtocol,
-    axes: Optional[Union[matplotlib.axes.Axes, Sequence[matplotlib.axes.Axes]]] = None,
-    colorbars: Optional[
-        Union[matplotlib.colorbar.Colorbar, Sequence[matplotlib.colorbar.Colorbar]]
-    ] = None,
+    axes: matplotlib.axes.Axes | Sequence[matplotlib.axes.Axes] | None = None,
+    colorbars: matplotlib.colorbar.Colorbar
+    | Sequence[matplotlib.colorbar.Colorbar]
+    | None = None,
```

Comment:
(no action needed) not sure what to think about this formatting, i think that i preferred what black does with long experessions inside brackets.

Reply:
Replaced by short form to avoid these overly long lines